utils/log/logger.go

```diff
@@ -22,6 +22,9 @@ var Format LogFormat
 // but through the 'isTerminalMode' function.
 var terminalMode *bool
 
+// DefaultLogger is the default logger instance in case the user does not set one
+var DefaultLogger = NewLogger(INFO, nil)
```

Comment:
It looks like DefaultLogger can be made private.
DefaultLogger --> defaultLogger